trade-finance-manager-ui/templates/utilisation-reports/utilisation-report-reconciliation-for-report.njk

```diff
@@ -152,7 +152,13 @@
 {% endset %}
 
 {% set paymentDetailsHtml %}
-  <div></div>
+  <h2 class="govuk-heading-l" data-cy="payment-details-heading">Payment details</h2>
+
+  <p class="govuk-body">
+    {% if paymentDetails.length === 0 %}
+      Payment details will be displayed when payments has been entered on the premium payments tab.
```

Comment:
Sorry for the small comment, could we update 'has' to 'have' please

The test that checks this message will also need updating (in `trade-finance-manager-ui/component-tests/utilisation-reports/utilisation-report-reconciliation-for-report.component-test.ts`)

```suggestion
      Payment details will be displayed when payments have been entered on the premium payments tab.
```